microscopic-monks/primal/engine/feature.py

```diff
@@ -0,0 +1,73 @@
+from typing import Tuple
+
+from primal.gui.health import HealthBar
+from primal.engine.sprite import RotatableSprite
+from kivy.graphics.instructions import InstructionGroup
+
+from kivy.storage.jsonstore import JsonStore
+
+store = JsonStore('inventory.py')
+
+
+class Feature:
+    def __init__(self, sprite: str, pos: Tuple[float, float], z: float, size: Tuple[float, float],
+                 angle: float, type: str, collide: bool = False):
+        hp_pos_x = (pos[0] + size[0] / 2) - 50
+        hp_pos_y = pos[1]
+
+        self.collide = collide
+        self.type = type
+        self.health_bar = HealthBar((hp_pos_x, hp_pos_y), (100, 7), 5)
```

Comment:
The use of positional arguments over keyword arguments makes it so it's hard to understand on first passing. 
If I want to know what the two next arguments are I would have to lookup HealthBar. I would love to see more keyword arguments in the future because arguments should be self-referencing.